src/layouts/components/Editor/LinkBubbleMenu.tsx

```diff
@@ -0,0 +1,110 @@
+import {
+  useDisclosure,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  InputGroup,
+  InputLeftAddon,
+} from "@chakra-ui/react"
+import { Button, Input } from "@opengovsg/design-system-react"
+import { BubbleMenu } from "@tiptap/react"
+import { useState } from "react"
+
+import { useEditorContext } from "contexts/EditorContext"
+import { useEditorModal } from "contexts/EditorModalContext"
+
+const LinkButton = () => {
+  const { editor } = useEditorContext()
+  const { onClose, onOpen, isOpen } = useDisclosure()
+  const { showModal } = useEditorModal()
+  const [href, setHref] = useState("")
+
+  return (
+    <>
+      <button
+        // TODO: Replace with a nice looking button
+        style={{
+          border: "1px solid black",
+          borderRadius: "5px",
+          padding: "1px 6px",
+          backgroundColor: "white",
+        }}
+        type="button"
+        onClick={() => {
+          // NOTE: If the link is an absolute link,
+          // it's not a file on Isomer so we will just allow them to change the link
+          // using the link modal.
+          if (
+            (editor.getAttributes("link").href as
+              | string
+              | undefined)?.startsWith("http")
+          ) {
+            onOpen()
+          } else {
+            // Otherwise, show the file modal
+            // and let the user select a file to link to.
+            showModal("files")
```

Comment:
What about relative links to other pages of the site?

Reply:
good catch. i'll add this in on another PR (the one where behaviour got changed) to avoid rebase errors